apps/src/componentLibrary/checkboxDropdown/checkboxDropdown.module.scss

```diff
@@ -0,0 +1,336 @@
+@import "color";
+@import "@cdo/apps/componentLibrary/common/styles/mixins";
+
+// Dropdown common styles
```

Comment:
Probably a follow-up once you've implemented all the dropdown variants, but how much of these styles can we factor out and share between the various dropdown components?

Reply:
Good question, plan is to factor out as much styles as possible in the process of working on `IconsDropdown`. For `SimpleDropdown` I feel like it's not applicable since though visually the same, semantically and by it's structutre they differ a lot.